src/api/spec/models/kiwi/repository_spec.rb

```diff
@@ -67,4 +67,19 @@
     it { is_expected.to allow_value(nil).for(:imageinclude) }
     it { is_expected.to allow_value(nil).for(:prefer_license) }
   end
+
+  context 'do not update with outdated image' do
+    let(:kiwi_repository) { create(:kiwi_repository_with_package) }
+    before do
+      allow_any_instance_of(Package).to receive(:kiwi_image_outdated?) { true }
```

Comment:
that means that every kiwi images needs a Package to get outdated, and I think that was not the original idea. So either we fix this, or we enforce that all kiwi image has a package (introducing a validation). We can do it in a different PR, but we should solve it.

@eduardoj @mdeniz @DavidKang ^^